python/DEV_SETUP.md

```diff
@@ -77,13 +77,42 @@ Poetry allows to use SK from the local files, without worrying about paths, as
 if you had SK pip package installed.
 
 To install Poetry in your system, first, navigate to the directory containing
-this README using your chosen shell. You will need to have Python 3.8+ installed.
+this README using your chosen shell. You will need to have Python 3.10, 3.11, or 3.12
+installed.
 
 Install the Poetry package manager and create a project virtual environment.
 Note: SK requires at least Poetry 1.2.0.
 
+### Note for MacOS Users
+
+It is best to install Poetry using their [official installer](https://python-poetry.org/docs/#installing-with-the-official-installer).
+
+On MacOS, you might find that `python` commands are not recognized by default, and you can only use `python3`. To make it easier to run `python ...` commands (which Poetry requires), you can create an alias in your shell configuration file.
```

Comment:
Are we limiting the length of the lines in the README file (looks like from the change in line 80)? If that's so, can we apply the same on this line?